components/list_item_group/list_item_group.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <ul
+    :id="id"
+    :class="['d-ps-relative', 'd-px0', listClass]"
+    data-qa="dt-dropdown-list-wrapper"
+    :aria-labelledby="`${id}-heading`"
+  >
+    <li
+      v-if="heading"
+      :id="`${id}-heading`"
+      role="presentation"
+      data-qa="dt-dropdown-list-heading"
+      :class="['dt-dropdown-list--header', headingClass]"
+    >
+      {{ heading }}
```

Comment:
I'd probably wrap this with a slot also in case we need to pass something more than just a string

Reply:
Yeah good idea.